modules/installation-launching-installer.adoc

```diff
@@ -159,6 +159,8 @@ You can run the `create cluster` command of the installation program only once,
 
 ifndef::osp,rhv,vsphere[* Configure an account with the cloud platform that hosts your cluster.]
 
+ifdef::rhv[* Open the `ovirt-imageio` port from the machine running the installer to the engine. By default, the port is `54322`.]
```

Comment:
```suggestion
ifdef::rhv[* Open the `ovirt-imageio` port to the Engine from the machine running the installer. By default, the port is `54322`.]
```
@ehvs I double-checked and it should be "Engine" with a capital "E". Sorry I didn't notice that before. And I suggest moving the clause "to the Engine" earlier in the sentence to make easier to parse.